cli/commands/configure.ts

```diff
@@ -12,19 +12,43 @@ export const configureCommand = (): CommandDefinition => ({
 			flags: '-d, --deleteBeforeUpdate',
 			description: `Useful when depending on the parameter having version 1, e.g. for use in CloudFormation`,
 		},
+		{
+			flags: '-X, --deleteParameters',
+			description: 'Deletes the parameters.',
```

Comment:
```suggestion
			flags: '-X, --deleteParameter',
			description: 'Deletes the parameter.',
```

It only deletes one parameter.